### ISSUE TITLE
About OFA-tiny zero-shot performance.

### ISSUE THREAD

**Issue by ZhangYuanhan-AI** [OFA-Sys/OFA-Compress#3]
Hello,

Thank you for providing a clear version of the OFA model.

<img width="844" alt="image" src="https://user-images.githubusercontent.com/18485270/227080663-24c750cf-6f56-4ea8-8668-9299a4dacc9a.png">

I have a question regarding the model displayed in the image above. Is this the OFA-tiny model, or is it the OFA-tiny model fine-tuned on Refcoco?

I assume it is the latter, as the OFA-tiny model exhibits poor performance during the zero-shot evaluation on Refcoco.

Thanks for your reply.

**Comment by faychu** [comment]
Yes, this is the result after finetuning the OFA-tiny model on Refcoco.

**Comment by ZhangYuanhan-AI** [comment]
Hi,

Thanks for your answering,

Another question is, what do these parameters mean?

"image_bucket_size" : https://github.com/OFA-Sys/OFA-Compress/blob/a8618ab5e4ed7613d69af80acec50eb0da9aa34d/ofa/configuration_ofa.py#L130
"token_bucket_size"

**Comment by ZhangYuanhan-AI** [comment]
Could you provide any insights regarding the rationale behind employing the "sum" of loss as a method during the pre-training phase?

https://github.com/OFA-Sys/OFA-Compress/blob/a8618ab5e4ed7613d69af80acec50eb0da9aa34d/adaptors.py#L51
